centreon/www/class/centreonGraph.class.php

```diff
@@ -251,12 +257,14 @@ public function __construct($user_id, $index = null, $debug = 0, $compress = nul
                 $this->onecurve = true;
             }
             $DBRESULT->closeCursor();
-            $DBRESULT = $this->DB->query("SELECT vmetric_id metric_id
-                                          FROM virtual_metrics
-                                          WHERE index_id = '" . $this->index . "'
-                                          AND ( `hidden` = '0' OR `hidden` IS NULL )
-                                          AND vmetric_activate = '1'
-                                          ORDER BY 'metric_name'");
+            $DBRESULT = $this->DB->prepare("SELECT vmetric_id metric_id
+              FROM virtual_metrics
+              WHERE index_id = :index_id
+              AND (`hidden` = '0' OR `hidden` IS NULL)
+              AND vmetric_activate = '1'
+              ORDER BY `metric_name`");
```

Comment:
The **metric_name** column does not exist.
```suggestion
              ORDER BY `vmetric_name`");
```